Dockerfile

```diff
@@ -44,17 +44,17 @@ COPY tests tests
 COPY tests_e2e tests_e2e
 COPY pytest.ini pytest.ini
 
-# Install Chrome
-RUN curl -sSL https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb -o chrome.deb &&\
+# Install Chrome (version 91.0.4472.77-1)
```

Comment:
I decided to fix the version of Chrome we use for Selenium tests, as every new version breaks the Travis build (this is due to the Chromium driver inevitably becoming out of date in the proceeding RUN statement).